modules/kms/README.md

```diff
@@ -1,40 +1,71 @@
 # Google KMS Module
 
-Simple Cloud KMS module that allows managing a keyring, zero or more keys in the keyring, and IAM role bindings on individual keys.
+This module allows creating and managing KMS keys and IAM bindings at both the keyring and key level. An existing keyring can be used, or a new one can be created and managed by the module if needed.
 
-The `protected` flag in the `key_attributes` variable sets the `prevent_destroy` lifecycle argument on an a per-key basis.
+When using an existing keyring be mindful about applying IAM bindings, as all bindings used by this module are authoritative, and you might inadvertently override bindings managed by the keyring creator.
+
+## Protecting against destroy
+
+In this module **no lifecycle blocks are set on resources to prevent destroy**, in order to allow for experimentation and testing where rapid `apply`/`destroy` cycles are needed. If you plan on using this module to manage non-development resources, **clone it and uncomment the lifecycle blocks** found in `main.tf`.
```

Comment:
I would mention that data / VM encrypted with a disabled/destroyed key won't be accessible once the crypto key / key version is deleted.

Reply:
I would assume whoever uses KMS understands what happens once you delete a key? :)